app/enrichment/create-invoice-number.js

```diff
@@ -11,6 +11,8 @@ const createInvoiceNumber = (paymentRequest) => {
         return createSitiAgriInvoiceNumber(paymentRequest)
       case FDMR:
         return createFdmrInvoiceNumber(paymentRequest)
+      case MANUAL:
+        return checkInvoiceNumber(paymentRequest)
```

Comment:
We could simplify this to just be:
```
case MANUAL:
  return paymentRequest.invoiceNumber
```

If there's no invoice number that would return `undefined` which would then fail the validation step against the joi schema after this.